regression/goto-instrument/restrict-function-pointer-to-multiple-functions-code-check/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.c
+--restrict-function-pointer use_f.function_pointer_call.1/f,g --show-goto-functions
+f\(10\)
```

Comment:
:green_heart: this test seems a precise approach

Reply:
What does the green heart mean?